[EDR Workflows] Correct bugs in Execute action API docs (#4236)
elastic/security-docs: docs/management/api/execute-api.asciidoc

|`case_ids` |Array (String) |The IDs of cases where the action taken will be logged. |No
|`comment` |String |Attach a comment to this action's log. The comment text will appear in associated cases. |No
|`parameters.command` |String |A shell command to run on the host. The command must be supported by `bash` for Linux and macOS hosts, and `cmd.exe` for Windows. |Yes

> |`parameters.timeout` |Integer |How long the host should wait for the command to complete in seconds.  |No

Reviewer: Alternate streamlined wording:
```suggestion
|`parameters.timeout` |Integer |The duration, in seconds, that the host waits for the command to complete.  |No
```

> |`parameters.timeout` |Integer |How long the host should wait for the command to complete in seconds.  |No

Reviewer 2: I guess it shouldn't, @paul-tavares @ashokaditya can you confirm that?

> |`parameters.timeout` |Integer |How long the host should wait for the command to complete in seconds.  |No

Reviewer 3: No the changed sentences for `timeout` do not affect the overall description of the parameter.

> |`parameters.timeout` |Integer |How long the host should wait for the command to complete in seconds.  |No

Reviewer 3: @nastasha-solomon if the `timeout` parameter is not specified at action input it defaults to a 4 hour (14400 seconds) duration
